hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/kms/KMSClientProvider.java

```diff
@@ -178,7 +178,7 @@ public static class KMSTokenRenewer extends TokenRenewer {
 
     @Override
     public boolean handleKind(Text kind) {
-      return kind.equals(TOKEN_KIND);
+      return kind.equals(KMSDelegationToken.TOKEN_KIND);
```

Comment:
Maybe HADOOP-19152 fixed this problem, before HADOOP-19152, `KMSClientProvider` inherited KeyProvider`, `KeyProvider` imported `org.bouncycastle.jce.provider.BouncyCastleProvider`, but `hadoop-client-runtime` did not package bcprov.

My change was to avoid initializing `KMSClientProvider`.

Do we need to backport HADOOP-19152 for 3.3 and 3.4?